node/dataSources/session.ts

```diff
@@ -22,19 +22,25 @@ export class SessionDataSource extends RESTDataSource<Context> {
     super()
   }
 
-  public getSegmentData = (defaultSegment: boolean = false) => this.get<SegmentData>('/segments', {defaultSegment})
+  public getSegmentData = (defaultSegment: boolean = false) =>
+    this.get<SegmentData>('/segments', { defaultSegment })
 
-  public getSession = () => this.get('/sessions/?items=*')
-  public updateSession = (key: string, value: any) => this.post('/sessions', { public: { [key]: { value } } })
+  public updateSession = (key: string, value: any) =>
+    this.post('/sessions', { public: { [key]: { value } } })
 
   get baseURL() {
-    const {vtex: {account}} = this.context
+    const {
+      vtex: { account },
+    } = this.context
     return `http://${account}.vtexcommercestable.com.br/api`
   }
 
-  protected willSendRequest (request: RequestOptions) {
+  protected willSendRequest(request: RequestOptions) {
     const defaultSegment = request.params.get('defaultSegment') === 'true'
-    const {cookies, vtex: {authToken}} = this.context
+    const {
+      cookies,
+      vtex: { authToken },
+    } = this.context
     const segment = !defaultSegment ? cookies.get('vtex_segment') : undefined
```

Comment:
you can get segment and session cookies in `{vtex: {segmentToken, sessionToken}}`